CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add Bash and PowerShell wrapper scripts.
+- Add `-wd` global parameter allowing to change the working directory.
```

Comment:
Consider add information about the new method: `Taskflow.WorkDirParam`.

The following line were added for `Taskflow.VerboseParam`:
> - Remove `TF.Verbose`, add `Taskflow.VerboseParam` instead.

Reply:
Tried to address 4f7ab5b40abd41ad5d433fce60f216ab542c3134